schema/paths/forms/types/form.yml

```
@@ -1,5 +1,11 @@
 components:
   schemas:
+    revision_id:
+      description: リビジョンID
```

```suggestion
      description: 改定番号
```
リビジョンIDだと英語のままなので、改定番号としたの方がいい気がするんですがどうでしょう？

---

改定...なんかかたっ苦しいです...履歴IDとかでいいんでは？要はgitのコミットSHAみたいなものなので

---

素直にバージョンでもいいかも

---

堅苦しいの確かに...
バージョンで良さそうですね👍👍